gcp_variant_transforms/testing/data/input_files/wrong

```diff
@@ -0,0 +1,3 @@
+./gcp_variant_transforms/testing/data/vcf/valid-4.0.vcf
```

Comment:
It is better to include this in unit tests (i.e., create temporary files there).

Reply:
Done.